File: Python/백준/Bronze/6436. Floppies/Floppies.py
import math

i=1
while(True):
    a = int(input())
    n = a/2*1.5/1860000
    if(a==0):
        break
    print(f'File #{i}')
    print(f'John needs {math.ceil(n)} floppies.')
    print()
    i+=1

File: Java/백준/Bronze/6436. Floppies/Floppies.java
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int i = 1;
        while(true){
            double a = in.nextDouble();
            if (a==0){
                break;
            }
            double b = Math.ceil(a/2480000);
            System.out.printf("File #%d\n",i);
            System.out.printf("John needs %.0f floppies.\n\n",b);
            i++;
        }
    }
}